docs/src/tutorials/npm-browser-packages/template-deep-dive/testing-your-project.md

```diff
@@ -0,0 +1 @@
+# Testing your project
```

Comment:
this looks like a stub that mdbook created - might mean your file is in the wrong directory?

Reply:
Oops yes, this shouldn't be here, deleted now